Makefile

```diff
@@ -1,17 +1,20 @@
+GOFILES=$(shell find . -name \*.go)
```

Comment:
That is not necessary. "go" does that for us. It just introduces another source for errors.